arbitrum-docs/for-devs/concepts/nodeinterface.mdx

```diff
@@ -1,11 +1,12 @@
 ---
-title: 'NodeInterface: A conceptual overview'
-sidebar_label: NodeInterface
+title: 'NodeInterface overview'
```

Comment:
Our two NodeInterface docs are now grouped in a NodeInterface folder, but we can probably merge them and place under Precompiles (or as a standalone doc if we want to make it more discoverable)

Reply:
Precompiles and the NodeInterface are slightly different, but I feel like readers associate them together many times. The only reason to separate them is to not confuse readers with their nature (precompiles are onchain and can be used by contracts; the NodeiIterface is not).
But it makes sense to have only one page with the short introduction and the reference below.